### ISSUE TITLE
cannot find redstorm gem when using rbenv

### ISSUE THREAD

**Issue by harrigan** [colinsurprenant/redstorm#1]
I followed the instructions in the README, but when I try the examples:

```
redstorm examples/local_exclamation_topology.rb
```

I get:

```
Exception in thread "main" java.lang.ExceptionInInitializerError
Caused by: org.jruby.exceptions.RaiseException: (LoadError) no such file to load -- red_storm/version
```

I am using OSX 10.6.8 and JRuby 1.6.5. Specifically, the error is caused by

```
java -cp "target/classes:target/dependency/*" redstorm.TopologyLauncher examples/local_exclamation_topology.rb
```

So JRuby can't find my installed gems. I am using rbenv (http://github.com/sstephenson/rbenv) so my gems are in a non-standard location. However, should the required gems also be in target/classes? For example, if I:

```
gem install -i target/classes redstorm ruby-maven --no-rdoc --no-ri
```

then the problem goes away.

**Comment by colinsurprenant** [comment]
I personnally use RVM and I never tested with rbenv... I will give it a try to see if I can reproduce your problem. Let me know if there is anything particular about your environment other than:
- install rbrnv
- install jruby 1.6.5 for rbenv
- intall RedStorm gem
- run RedStorm example

**Comment by harrigan** [comment]
I installed rbenv, JRuby 1.6.5 and the RedStorm gem on a different machine (Ubuntu 10.04.2 LTS) and I was able to reproduce the problem.

**Comment by colinsurprenant** [comment]
reproduced the problem on OSX. Working on it!

**Comment by colinsurprenant** [comment]
ok, found the source of the problem. GEM_HOME is not set by rbenv. I am trying to figure how to work around this in the bootstrap code, in the meanwhile, you can manually set GEM_HOME and it should work. For me this was:

$ export GEM_HOME=~/.rbenv/versions/jruby-1.6.5/lib/ruby/gems/1.8/

please let me know if by setting your GEM_HOME it works for you.

**Comment by harrigan** [comment]
Yes, setting GEM_HOME fixes the error.

**Comment by colinsurprenant** [comment]
ok, fixed in v0.1.1. gem will be pushed shortly.

**Comment by adamgotterer** [comment]
I just ran into this same problem on the latest version of redstorm. Is there an up to date work around?
